### DISCUSSION TITLE
Improve our logo

### DISCUSSION THREAD

**Discussion by kiwicopple** [supabase/supabase#194] [General]
This is a Hacktoberfest task! 
You can read more about Hacktoberfest [on our blog](https://supabase.io/blog/2020/09/11/supabase-hacktoberfest-2020) and see all tasks on our [Hacktoberfest Project Board](https://github.com/orgs/supabase/projects/5).

# Summary

Don't like our logo or think you can do better? Why not give it a go? Add your ideas as a comment below!

![image](https://user-images.githubusercontent.com/10214025/93011810-4c45d580-f5cc-11ea-8f83-2ef7b5bb671f.png)

## Details

Some ideas:

* Complete redesign
* Create a Supabase icon
* Create a logo animation
* Tweaks and improvements

**Comment by royquilor** [comment]
Hi, please could you provide up to 3 keywords that represent the brand?

**Comment by steve-chavez** [comment]
Hey @royquilor. My take on the keywords: `PostgreSQL`, `Real-time`, `No-Code`.

I stole them from the [Supabase landing page](https://supabase.io/) :)

**Comment by kiwicopple** [comment]
Some potential non-technical terms: `simple`, `fast`, `powerful`

**Comment by royquilor** [comment]
Hi all, explored some concepts with simple, fast and powerful. Tested in a smaller size. Look forward to your feedback.

Concept A:
![supabase-concept-a](https://user-images.githubusercontent.com/2366186/96035184-9388e580-0e5a-11eb-923b-fb085fd3224d.png)

Concept B:
![supabase-concept-b](https://user-images.githubusercontent.com/2366186/96035216-9f74a780-0e5a-11eb-8054-82379a2d2ce7.png)

**Comment by kiwicopple** [comment]
Wow this is awesome @royquilor - I particularly appreciate you adding the smaller versions so that I can see how they scale. I think the top one scales a lot better, though I do prefer the bottom left. Perhaps one improvement would be if the lines were all the same width (my opinion). 

Are you participating in Hacktoberfest? If so, you can make this a "pull request" and I can accept it?

Next steps, we'll wait until the end of the month and see if any other Designers want to contribute their own ideas - maybe even riff off the work you've already done.

**Comment by kiwicopple** [comment]
From @nancu : https://github.com/supabase/supabase/pull/263

![image](https://user-images.githubusercontent.com/10214025/97135195-9c4ab700-178a-11eb-96b9-a7b731d934cb.png)
![image](https://user-images.githubusercontent.com/10214025/97135203-a076d480-178a-11eb-811e-9e10fe9b258b.png)

**Comment by kiwicopple** [comment]
Since we are outside of Hacktoberfest, I've converted this to a discussion. This will make it easier to add examples (copy/paste rather than PR)
